app/Models/IndicatorStatus.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Models;
+
+use Backpack\CRUD\app\Models\Traits\CrudTrait;
+use Illuminate\Database\Eloquent\Model;
+
+class IndicatorStatus extends Model
+{
+    use CrudTrait;
+
+    /*
+    |--------------------------------------------------------------------------
+    | GLOBAL VARIABLES
+    |--------------------------------------------------------------------------
+    */
+
+    protected $table = 'indicators_status';
+    // protected $primaryKey = 'id';
+    // public $timestamps = false;
+    protected $guarded = ['id'];
+    // protected $fillable = [];
+    // protected $hidden = [];
+    // protected $dates = [];
+
+    /*
+    |--------------------------------------------------------------------------
+    | FUNCTIONS
+    |--------------------------------------------------------------------------
+    */
+
+    /*
+    |--------------------------------------------------------------------------
+    | RELATIONS
+    |--------------------------------------------------------------------------
+    */
+    public function indicator_values()
+    {
+        return $this->belongsTo('App\Models\IndicatorValue');
+    }
```

Comment:
Is the relationship between IndicatorStatus and IndicatorValue the right way round? Without seeing the schema, I would normally assume that an IndicatorValue belongs to one status, and one status has many values?